app/protocol/flows/blockrelay/handle_relay_invs.go

```diff
@@ -118,10 +122,13 @@ func (flow *handleRelayInvsFlow) readInv() (*appmessage.MsgInvRelayBlock, error)
 }
 
 func (flow *handleRelayInvsFlow) requestBlocks(requestQueue *hashesQueueSet) error {
+	log.Tracef("requestBlocks start")
```

Comment:
Here too: logger.LogAndMeasureExecutionTime.
Unless you specifically want it to be trace, and then maybe better add logger.LogAndMeasureExecutionTimeTrace, or make it receive log level, or something.